oggm/core/massbalance.py

```diff
@@ -915,6 +915,7 @@ def __init__(self, gdir, fls=None, mu_star=None,
                                    'use_inversion_flowlines=True.') from None
 
         self.fls = fls
+        _y0 = kwargs.get('y0')
```

Comment:
`kwargs.get('y0', None)` to set a default value in case y0 isn't provided at all (otherwise this might raise a ValueError

Reply:
`dict.get()` defaults to `None` if the key is not found. But yes, explicit is better than implicit ;-).